### PR TITLE
chore: Avoid warning for CoreApplication.Exit

### LINKED ISSUE #10436: [Skia] Implement `Application.Exit`
### Discussed in https://github.com/unoplatform/uno/discussions/10404

<div type='discussions-op-text'>

<sup>Originally posted by **mcNets** November 11, 2022</sup>
Exit method is working now, it was solved in this PR https://github.com/unoplatform/uno/pull/10001

But compiler still throwing a warning:

`warning Uno0001: Microsoft.UI.Xaml.Application.Exit() is not implemented in Uno`</div>

## Additional details

This method is not implemented at this time, but `CoreApplication.Exit` was implemented in  https://github.com/unoplatform/uno/pull/10001. Implementing `Application.Exit` is about calling one from the other.

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #10436

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿#nullable enable
+
+namespace Windows.ApplicationModel.Core;
+
+partial class CoreApplication
+{
+	private static void ExitPlatform()
+	{
+	}
```
Comment (Youssef1313) [inline src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs:9]: I don't think an empty implementation like that is correct? Also reported issue for Skia so `CoreApplication.Exit` is already implemented.

Regarding `Application.Exit`:

https://github.com/unoplatform/uno/blob/c4288e1062bda783397d2b3145086298883c73d0/src/Uno.UI/UI/Xaml/Application.cs#L231-L245

It's also already implemented in Skia and delegates to `CoreApplication.Exit` and should be working already. (the original issue reported in fact says that it's working)

I think the issue might be that we don't pass specific platforms to `NotImplemented` attribute. I don't recall why the `#if` directives alone are not sufficient though.

Comment (Youssef1313) [reply src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs:9]: To my understanding, the fix should be a single line changing the following line:

https://github.com/unoplatform/uno/blob/c4288e1062bda783397d2b3145086298883c73d0/src/Uno.UI/UI/Xaml/Application.cs#L232

to

```csharp
[NotImplemented("__IOS__", "NET461", "__WASM__", "__NETSTD_REFERENCE__")]
```

Comment (Youssef1313) [reply src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs:9]: I went ahead and opened https://github.com/unoplatform/uno/pull/10457

Comment (MartinZikmund) [reply src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs:9]: You are right, the reported warning was most likely caused by that `[NotImplemented]` attribute! I think this one is not needed then.

**Pull request closed without merging**